diff --git a/node_modules/react-native-paper/src/styles/themes/v3/DarkTheme.tsx b/node_modules/react-native-paper/src/styles/themes/v3/DarkTheme.tsx
index 19651a0..3186a5a 100644
--- a/node_modules/react-native-paper/src/styles/themes/v3/DarkTheme.tsx
+++ b/node_modules/react-native-paper/src/styles/themes/v3/DarkTheme.tsx
@@ -5,7 +5,6 @@ import { MD3LightTheme } from './LightTheme';
 import { MD3Colors, tokens } from './tokens';
 
 const { palette, opacity } = tokens.md.ref;
-
 export const MD3DarkTheme: MD3Theme = {
   ...MD3LightTheme,
   dark: true,
diff --git a/node_modules/react-native-paper/src/styles/themes/v3/LightTheme.tsx b/node_modules/react-native-paper/src/styles/themes/v3/LightTheme.tsx
index a897eff..ba6ba1f 100644
--- a/node_modules/react-native-paper/src/styles/themes/v3/LightTheme.tsx
+++ b/node_modules/react-native-paper/src/styles/themes/v3/LightTheme.tsx
@@ -3,11 +3,10 @@ import color from 'color';
 import type { MD3Theme } from '../../../types';
 import configureFonts from '../../fonts';
 import { MD3Colors, tokens } from './tokens';
-
 const { palette, opacity } = tokens.md.ref;
 
 export const MD3LightTheme: MD3Theme = {
-  dark: false,
+  dark: true,
   roundness: 4,
   version: 3,
   isV3: true,
@@ -15,7 +14,7 @@ export const MD3LightTheme: MD3Theme = {
     primary: palette.primary40,
     primaryContainer: palette.primary90,
     secondary: palette.secondary40,
-    secondaryContainer: palette.secondary90,
+    secondaryContainer: 'rgba(255, 255, 255, 0)',
     tertiary: palette.tertiary40,
     tertiaryContainer: palette.tertiary90,
     surface: palette.neutral99,
